src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -722,6 +722,76 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="Error_NoMatchingCertificate" xml:space="preserve">
     <value>The package signature did not match any of the allowed certificate fingerprints.</value>
   </data>
+  <data name="Error_CannotTrustOwnersForAuthor" xml:space="preserve">
+    <value>Trusted owners are not supported in trusted author items.</value>
+  </data>
+  <data name="Error_InvalidCombinationOfArguments" xml:space="preserve">
+    <value>Invalid combination of arguments.</value>
+  </data>
+  <data name="Error_NoSignatureTrustedForPackage" xml:space="preserve">
+    <value>No signature was specified to be trusted for package '{0}'. Please specify either Author or Repository.</value>
+    <comment>0 - Package path</comment>
+  </data>
+  <data name="Error_NoSourcesInformation" xml:space="preserve">
+    <value>Unable to get package sources information.</value>
+  </data>
+  <data name="Error_NotSupportedHashAlgorithm" xml:space="preserve">
+    <value>'{0}' is not a supported hash algorithm.</value>
+    <comment>0 - hash algorithm</comment>
+  </data>
+  <data name="Error_PropertyCannotBeNullOrEmpty" xml:space="preserve">
+    <value>Property '{0}' should not be null or empty.</value>
+    <comment>0 - property name</comment>
+  </data>
+  <data name="Error_ServiceIndexShouldBeHttps" xml:space="preserve">
+    <value>The repository service index '{0}' is not a valid HTTPS url.</value>
+    <comment>0 - service index</comment>
+  </data>
+  <data name="Error_UnavailableSource" xml:space="preserve">
+    <value>Source with name '{0}' does not exist or cannot be retrived.</value>
+    <comment>0 - source name</comment>
+  </data>
+  <data name="NoTrustedSigners" xml:space="preserve">
+    <value>There are no trusted signers.</value>
+  </data>
+  <data name="NoTrustedSignersMatching" xml:space="preserve">
+    <value>No trusted signers with name: '{0}' where found.</value>
+    <comment>0 - trusted signer name</comment>
+  </data>
+  <data name="RegsiteredTrustedSigners" xml:space="preserve">
+    <value>Registered trusted signers:</value>
+  </data>
+  <data name="SuccessfullyRemovedTrustedSigner" xml:space="preserve">
+    <value>Successfully removed trusted signers with name: '{0}'.</value>
+    <comment>0 - trusted signer name</comment>
+  </data>
+  <data name="TrustedSignerLogCertificates" xml:space="preserve">
+    <value>Certificate fingerprint(s):</value>
+  </data>
+  <data name="TrustedSignerLogCertificateSummaryAllowUntrustedRoot" xml:space="preserve">
+    <value>[U] {0} - {1}</value>
```

Comment:
What should they localized here? :D 
Puta comment that there's nothing to be localized.